packages/module/src/components/VncConsole/VncActions.tsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
-import { css } from '@patternfly/react-styles';
 import { Button, ButtonVariant } from '@patternfly/react-core';
 import { Dropdown, DropdownItem, DropdownList, MenuToggle, MenuToggleElement } from '@patternfly/react-core';
 
-import styles from '@patternfly/react-styles/css/components/Consoles/VncConsole';
+import clsx from 'clsx';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+  consoleVnc: {
+    gridArea: 'main'
+  },
+  consoleActionsVnc: {
+    gridArea: 'actions-extra',
+    display: 'flex',
```

Comment:
I wonder if we can use a flex layout here instead of all these flex related css overrides

Reply:
Makes sense we should add this.  Created an issue for this so that we can add this without slowing down the release as this will require some rework (https://github.com/patternfly/react-console/issues/62).